client/my-sites/checkout/src/components/akismet-pro-quantity-dropdown/index.tsx

```diff
@@ -214,32 +292,20 @@ export const AkismetProQuantityDropDown: FunctionComponent< AkismetProQuantityDr
 					open={ open }
 					role="button"
 				>
-					{ translate( '%(count)d Site', '%(count)d Sites', {
-						count: selectedQuantity,
-						args: {
-							count: selectedQuantity,
-						},
-					} ) }
+					{ dropdownOptions[ selectedQuantity - 1 ].label }
 					<Gridicon icon={ open ? 'chevron-up' : 'chevron-down' } />
 				</CurrentOption>
 				{ open && (
 					<OptionList role="listbox" tabIndex={ -1 }>
-						{ dropdownOptions.map( ( value ) => (
+						{ dropdownOptions.map( ( { id, label } ) => (
```

Comment:
Analogically, here we can use native index
```suggestion
						{ dropdownOptions.map( ( { label }, index ) => (
```

Reply:
Yes, done. 👍